src/tiptap-editor.vue

```diff
@@ -385,8 +382,10 @@ export default {
             if (this.editor) {
                 // preserve selection after updating warnings
                 const oldSelection = this.editor.view.state.selection;
-                this.editor.commands.setContent(this.currentValue);
```

Comment:
I think we want to leave this in the codebase for now, behind a comment so that it's disabled.  I think we'll need to come back to this to fix it or to remove the watcher entirely in the near future.